incubator/faucet/alias.go

```diff
@@ -1,8 +1,8 @@
 package faucet
```

Comment:
nit: you can remove this file. It doesn't serve a purpose in the stargate release